### PR TITLE
Update to SDK 0.11.0

### PR DESCRIPTION
**Pull request by anuraaga**

### CODE REVIEW

File: instrumentation/aws-lambda-1.0/library/src/test/groovy/io/opentelemetry/instrumentation/awslambda/v1_0/AwsLambdaSqsMessageHandlerTest.groovy [hunk 1]
```diff
@@ -5,19 +5,19 @@
...
 import io.opentelemetry.api.trace.attributes.SemanticAttributes
 import io.opentelemetry.api.trace.propagation.HttpTraceContext
 import io.opentelemetry.context.propagation.DefaultContextPropagators
 import io.opentelemetry.extension.trace.propagation.AwsXRayPropagator
+import io.opentelemetry.instrumentation.test.AgentTestRunner
 import io.opentelemetry.instrumentation.test.InstrumentationSpecification
 import io.opentelemetry.instrumentation.test.InstrumentationTestTrait
 
+import static io.opentelemetry.api.trace.Span.Kind.CONSUMER
+import static io.opentelemetry.api.trace.Span.Kind.SERVER
```
Comment (mateuszrzeszutek) [inline instrumentation/aws-lambda-1.0/library/src/test/groovy/io/opentelemetry/instrumentation/awslambda/v1_0/AwsLambdaSqsMessageHandlerTest.groovy:19]: Shouldn't static imports go before "normal" imports?

File: testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/AgentTestRunner.java [hunk 2]
```diff
@@ -109,9 +112,17 @@
...
+          OpenTelemetrySdk.builder()
+              .setResource(OpenTelemetrySdk.get().getResource())
+              .setClock(OpenTelemetrySdk.get().getClock())
+              .setMeterProvider(OpenTelemetry.getGlobalMeterProvider())
+              .setTracerProvider(unobfuscate(OpenTelemetry.getGlobalTracerProvider()))
+              .setPropagators(
+                  DefaultContextPropagators.builder()
+                      .addTextMapPropagator(HttpTraceContext.getInstance())
+                      .build())
               .build());
```
Comment (mateuszrzeszutek) [inline testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/AgentTestRunner.java:126]: Minor reuse suggestion:

```suggestion
      setGlobalPropagators(DefaultContextPropagators.builder()
              .addTextMapPropagator(HttpTraceContext.getInstance())
              .build());
```

**Review by jkwatson — APPROVED** [review]
👍

File: instrumentation/aws-lambda-1.0/library/src/test/groovy/io/opentelemetry/instrumentation/awslambda/v1_0/TracingRequestStreamWrapperTest.groovy [hunk 3]
```diff
@@ -5,20 +5,21 @@
...
 import io.opentelemetry.context.propagation.DefaultContextPropagators
+import io.opentelemetry.instrumentation.test.AgentTestRunner
 import io.opentelemetry.instrumentation.test.InstrumentationSpecification
 import io.opentelemetry.instrumentation.test.InstrumentationTestTrait
-import java.nio.charset.Charset
 import org.junit.Rule
 import org.junit.contrib.java.lang.system.EnvironmentVariables
 import spock.lang.Shared
 
+import java.nio.charset.Charset
```
Comment (trask) [inline instrumentation/aws-lambda-1.0/library/src/test/groovy/io/opentelemetry/instrumentation/awslambda/v1_0/TracingRequestStreamWrapperTest.groovy:19]: new Intellij workspace? looks like needs https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/docs/contributing/intellij-setup.md

**Review by trask — APPROVED** [review]

**Comment by trask** [comment]
@anuraaga I caused merge conflicts for this PR, so cleaned them up (and fixed a test). Our groovy imports have drifted a bit (since we have no way to enforce) so I'll probably send separate PR after this to clean them all up.

**Comment by anuraaga** [comment]
Thanks - sorry didn't reconfigure IntelliJ on my new computer. Since we're not spotlessing, I wonder if we should remove the steps from https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/docs/contributing/intellij-setup.md and make it as close to the default experience as possible.

**Pull request merged**